docs/dataset_prepare.md

```diff
@@ -253,3 +253,31 @@ Since we only support test models on this dataset, you may only download [the va
 ### Nighttime Driving
 
 Since we only support test models on this dataset, you may only download [the test set](http://data.vision.ee.ethz.ch/daid/NighttimeDriving/NighttimeDrivingTest.zip).
+
+
+### LoveDA
+The data could be downloaded [here](https://drive.google.com/drive/folders/1ibYV0qwn4yuuh068Rnc-w4tPi0U0c-ti?usp=sharing)
+
+For LoveDA dataset, please run the following command to download and re-organize the dataset.
+
+```shell
+# download
+mkdir loveda && cd loveda
+wget https://drive.google.com/drive/folders/1ibYV0qwn4yuuh068Rnc-w4tPi0U0c-ti?usp=sharing
+
+# unzip
+unzip \*.zip
```

Comment:
Should it be `unzip ./*.zip` or something else?

Reply:
The '\' refers to the inversion symbol. This can help unzip all the zips.